### PR TITLE
Sketch out the 0.22.13 release note

### LINKED ISSUE #6331: EOL for the `0.22` series
It has been discussed multiple times in different PRs, so I think it's worth opening the issue and making this process transparent. What stops us from 'archiving' the 0.22 series at this point? I guess we have to stop updating 0.22 through Scala Steward. And we have to remove some update ignorings in the config (that we are getting from the 0.22 series now). Anything else?

### PR DESCRIPTION
**Pull request by danicheg**
This (finally) closes #6331.

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -4,6 +4,77 @@
...
+    * Update scala-xml to 2.1.0 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6234
+    * Update scalafmt-core to 3.5.1 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6256
+    * Update sbt-scalafix, scalafix-core, ... to 0.10.0 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6260
+    * Update Java-WebSocket to 1.5.3 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6270
+    * Fix deprecated since versions by @danicheg in https://github.com/http4s/http4s/pull/6279
+    * Remove ignoring some files for `doctest` by @danicheg in https://github.com/http4s/http4s/pull/6284
+    * Update sbt-http4s-org to 0.13.2 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6287
+    * Update netty-buffer, netty-codec-http to 4.1.76.Final in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6292
+    * Update scalafmt-core to 3.5.2 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6318
+    * Update fs2-core, fs2-io, ... to 2.5.11 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6322
```
Comment (armanbilge) [inline docs/changelog.md:70]: I think Ross likes to mention some of the important update in the main notes e.g. see:
https://github.com/http4s/http4s/releases/tag/v0.23.11

So probably for core should mention fs2 and cats-parse.

Also should mentioned tomcat, jetty, scala-xml updates for their modules.

**Review by armanbilge — COMMENTED** [review]
Thank you so much for spear-heading this release ❤️

File: docs/changelog.md (hunk 1 above)
Comment (danicheg) [reply docs/changelog.md]: I've polished this a bit up.

**Review by armanbilge — APPROVED** [review]
LGTM! 😀

**Comment by armanbilge** [comment]
0.22 is now officially EOL: https://http4s.org/versions.html

**Comment by danicheg** [comment]
It's a happy and a bit sad moment same time. Probably it should be announced on Twitter.

**Review by hamnis — APPROVED** [review]

**Comment by danicheg** [comment]
@rossabaker Ross, if you have resources to check this out it'd be cool. We've warmed up the engine and are ready to release and finally EOL the 0.22.

**Comment by rossabaker** [comment]
I'll work on those next, and we can put a bow on the whole stack.

**Comment by armanbilge** [comment]
> It might make sense to do a final patch release of all the CE2 dependencies and get them included.

We got in the latest fs2 2.x already. It seems we could use another CE2 release however with https://github.com/typelevel/cats-effect/pull/2420 and https://github.com/typelevel/cats-effect/pull/2593.

**Comment by rossabaker** [comment]
* [ ] unique
* [ ] vault
* [ ] log4cats
* [ ] keypool
* [ ] cats-effect
* [ ] fs2

**Comment by danicheg** [comment]
So, it seems we are good to finish that.

**Comment by rossabaker** [comment]
Yep.  I have soccer, but I can update the release notes and start the process.

Am I correct that we don't need to do that reformatting in Modern Laika?

**Comment by armanbilge** [comment]
As far as I understand, it should be possible to use `#` and `##` etc. as usual now.

**Comment by rossabaker** [comment]
Or anyone can turn the crank as well as me... if @danicheg wants to finish what he started, that's cool too.  I'm going to be AFK until late tonight.

**Comment by danicheg** [comment]
I could do it, just tomorrow. In my area, it's 2 am :)

**Comment by rossabaker** [comment]
In my area, 2am is when the kids are in bed and the dogs are asleep and I can get some work done!

**Comment by danicheg** [comment]
So, I have sketched a draft of the release (https://github.com/http4s/http4s/releases/edit/untagged-e75c2f9136dfd57bb597) and have freshened site release notes. I need someone with a steadfast gaze and the power to press the release button.

**Pull request merged**